### DISCUSSION TITLE
Error while running peridigm

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#344] [Q&A]
While running peridigm this error is showing, please help me to resolve this issue. is it because of dense meshing, I am using 900000 elements in my sample and using tet meshing. 
![WhatsApp Image 2023-07-04 at 18 20 28](https://github.com/peridigm/peridigm/assets/110670051/ff6785c7-ac32-4c0b-ba77-d7e08ecd750c)

**Comment by djlittl** [comment]
The simplest explanation is that the file can't be opened for some reason.  Any chance there is a typo in your input deck, for example can you confirm that the file name is spelled correctly and the file is in the correct directory?

Is this the entire error message, or is there more text off the righthand side of the screen?

900,000 elements should be fine.  There's a chance that you'd want/need to run in parallel.
